### ISSUE TITLE
simple_url_preview.dart error after updateing to a new flutter version

### ISSUE THREAD

**Issue by zalekbloom** [Amitbhave/flutter-simple-url-preview#26]
After updating to a new flutter version I got the error:
C:/Users/hayor/AppData/Local/Pub/Cache/hosted/pub.dev/simple_url_preview-3.0.1/lib/**simple_url_preview.dart:**174:54: Error: The getter 'accentColor' isn't defined for the class 'ThemeData'.

I am not expert in flutter but this site: 
https://docs.flutter.dev/release/breaking-changes/theme-data-accent-properties#migration-guide 
says:

Code before migration:
MaterialApp(
  theme: ThemeData(**accentColor**: myColor),
  // ...
);
Code after migration:

content_copy
final ThemeData theme = ThemeData();
MaterialApp(
  theme: theme.copyWith(
    colorScheme: theme.colorScheme.copyWith(secondary: myColor),
  ),
  //...
)

It looks that 'accentColor' is not defined there anymore.

But maybe I am wrong - can you look into this problem?

Thanks,
zb

**Comment by Amitbhave** [comment]
Yes, you are right. The plugin is not yet migrated to Flutter 3.0. We will have to migrate to 3.0.

**Comment by zalekbloom** [comment]
Can you let me know when your migration is ready?

Thanks,

zb
________________________________

**Comment by EtcGonza** [comment]
Same issue here.
